flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaSinkBuilderTest.java

```diff
@@ -77,6 +77,15 @@ public void testPropertyHandling() {
                     assertEquals("v1", p.get("k1"));
                     assertEquals("correct", p.get("k2"));
                 });
+
+        Properties testConf1 = new Properties();
```

Comment:
It would be better to have a separate test case for this because it adds new functiontionality.

Reply:
done.